lib/createStylelintResult.js

```diff
@@ -1,5 +1,5 @@
-/* @flow */
 "use strict"
+/* @flow */
```

Comment:
We've been putting the Flow pragma *above* `'use strict', right? Why move it here?